docs/semconv/database.md

```diff
@@ -261,6 +265,36 @@ This metric is [optional](https://github.com/open-telemetry/semantic-conventions
 |--------------------------|-----------------|------------------|----------------------------------------------------------------------------------------------|
 | `db.overflow.user.count` | UpDownCounter   | `{overflowUser}` | Number of times a User thread attempted to acquire a lock when no locks were available.      |
 
+### Metric: `db.cache.read.ratio`
+This metric is [optional](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metric-requirement-level.md#opt-in).
+
+| Name                  | Instrument Type   | Units (UCUM)       | Description                                                                              |
+|-----------------------|-------------------|--------------------|------------------------------------------------------------------------------------------|
+| `db.cache.read.ratio` | UpDownCounter     | `{cacheReadRatio}` | Percentage of page reads for this buffer pool that were satisfied by a cached page image |
+
+
+## Metric: `db.cache.write.ratio`
```

Comment:
Please use level 3 headings instead of level 2. Change to `### Metric: `db.cache.write.ratio`.